src/screens/Search/FilterSub.js

```diff
@@ -69,10 +69,14 @@ class FilterSub extends Component {
     return (
       <View style={{ padding: 20, paddingTop: 40, flexDirection: 'column' }}>
         <View style={{ flexDirection: 'row' }}>
-          <Left><Text style={styles.outerText}>Start: {startV}</Text></Left>
+          <Left>
+            <Text style={{...styles.outerText, paddingLeft: 20}}>
+              {startV}
```

Comment:
This will make it display just the start value instead of `Start: <value>`.

Reply:
That was the intention.